src/include/c_api/kuzu.h

```diff
@@ -2,6 +2,9 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <time.h>
+
+#include <string>
```

Comment:
`kuzu.h` is a C header. not a C++ header. It is a binding to C++ with [opaque pointer pattern](https://en.wikipedia.org/wiki/Opaque_pointer). You should NOT import any C++ header into this file at all. `<string>` is a C++ header, so it is not allowed in this file.